lib/ActivityStreams.js

```diff
@@ -202,15 +202,14 @@ ActivityStreams.prototype = {
    */
   _processAndSendLinks(links, responseType, worker, options) {
```

Comment:
Can we rename links to placesLinks so it's clear here exactly which ones we're talking about